### ISSUE TITLE
provide example SQL tables for relational data-stores

### ISSUE THREAD

**Issue by rjrodger** [senecajs/seneca-user#13]

**Comment by sorin7486** [comment]
Yes please. Could this also be solved in the data-store plugins? I ran into it while trying to use the mysql-store so I submitted a bug there:

https://github.com/mirceaalexandru/seneca-mysql-store/issues/13
